docs/backends/jaeger.mdx

```diff
@@ -1,34 +1,34 @@
 ---
-title: "Jaeger"
+title: 'Jaeger'
 ---
 
 ## Configuring Jaeger Backend
 
 Version v1.35 of Jaeger introduced the ability to receive OpenTelemetry trace data via the OpenTelemetry Protocol (OTLP).
 This allows to create a new Jaeger backend by simply specifying the Jaeger OTLP gRPC unencrypted URL.
 
-The endpoint format is `host:port`. 
-- host is required
-- port is optional and defaults to the default OTLP gRPC port `4317`.
+- **JAEGER_URL** - Endpoint, the format is `host:port`.
 
+  - host is required
+  - port is optional and defaults to the default OTLP gRPC port `4317`.
 
-## Adding a Destination to Odigos
+- **JAEGER_CERT_PEM** - Client certificate in PEM format. This is optional and only required if the Jaeger backend requires client authentication (TLS).
 
-Odigos makes it simple to add and configure destinations, allowing you to select the specific signals [traces/logs/metrics] that you want to send to each destination. There are two primary methods for configuring destinations in Odigos:
+- **JAEGER_KEY_PEM** - Client private key in PEM format. This is optional and only required if the Jaeger backend requires client authentication (TLS).
```

Comment:
In the UI it will not be `JAEGER_CERT_PEM`. Should we use the display name here for better user experience?